otel/otel.go

```diff
@@ -79,6 +98,18 @@ func Init(ctx context.Context) func() error {
 		hostName = setHostname()
 	}
 
+	defaultOptions := &initDefaultOptions{
+		defaultAttributes: []attribute.KeyValue{
+			semconv.ServiceName(cfg.ServiceName),
+			semconv.ServiceInstanceID(serviceInstanceID),
+			semconv.HostName(hostName),
```

Comment:
Can you set back the doc url, it is kind of useful
```suggestion	
            // https://opentelemetry.io/docs/specs/semconv/resource/host/
			semconv.HostName(hostName),
```

Reply:
Done